### DISCUSSION TITLE
Portable to the Cataclysm preservation project ? Wish it was :)

### DISCUSSION THREAD

**Discussion by Massattack82** [trickerer/Trinity-Bots#839] [Ideas]
anyway to Port this over to the Cataclysm preservation project ?  Would be nice it get bots past WOTLK :) Emucoach did it but not so great .. hopping to try myself but I'm not a coder :)

**Comment by trickerer** [comment]
About other expansions support
https://github.com/trickerer/Trinity-Bots/discussions/361
https://github.com/trickerer/Trinity-Bots/issues/472
Also, https://github.com/trickerer/Trinity-Bots/discussions/336 (it's in Russian, please use a translator)
